scripts/missions/windurst/8_2_The_Jester_Whod_be_King.lua

```diff
@@ -13,7 +13,7 @@
 -- Tosuka-Porika    : !pos -26 -6 103 238
 -- Kupipi           : !pos 2 0.1 30 242
 -- Shantotto        : !pos 122 -2 112 239
--- _5e5 (Cr. Wall)  : !pos -424.255 -1.909 619.995 194
+-- _5e5 (Cr. Wall)  : !pos -423 0 619.75 194
```

Comment:
```sql

INSERT INTO `npc_list` VALUES (17572260,'_5e5','Cracked Wall',0,-424.255,-1.909,619.995,1,40,40,9,0,0,0,3,0x0200000000000000000000000000000000000000,0,NULL,0);
```
its the exact pos in our database

Reply:
yeah, this change puts you in front of the npc instead of inside it.

Reply:
we're normally just listing where the NPC is here, can just wallhack out of the npc, since if using it obviously is a GM. but its _technically not wrong_